[2022-06-17 07:40:56,869] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: spark-test.spark_job manual__2022-06-17T07:40:54.719800+00:00 [queued]>
[2022-06-17 07:40:56,880] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: spark-test.spark_job manual__2022-06-17T07:40:54.719800+00:00 [queued]>
[2022-06-17 07:40:56,881] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-17 07:40:56,882] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-17 07:40:56,883] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-17 07:40:56,900] {taskinstance.py:1377} INFO - Executing <Task(SparkSubmitOperator): spark_job> on 2022-06-17 07:40:54.719800+00:00
[2022-06-17 07:40:56,906] {standard_task_runner.py:52} INFO - Started process 550 to run task
[2022-06-17 07:40:56,909] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'spark-test', 'spark_job', 'manual__2022-06-17T07:40:54.719800+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/spark-test.py', '--cfg-path', '/tmp/tmpd28tog48', '--error-file', '/tmp/tmpxvsyg55j']
[2022-06-17 07:40:56,910] {standard_task_runner.py:80} INFO - Job 3: Subtask spark_job
[2022-06-17 07:40:56,966] {task_command.py:370} INFO - Running <TaskInstance: spark-test.spark_job manual__2022-06-17T07:40:54.719800+00:00 [running]> on host f596236ec958
[2022-06-17 07:40:57,044] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@***.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=spark-test
AIRFLOW_CTX_TASK_ID=spark_job
AIRFLOW_CTX_EXECUTION_DATE=2022-06-17T07:40:54.719800+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-17T07:40:54.719800+00:00
[2022-06-17 07:40:57,052] {base.py:68} INFO - Using connection ID 'spark_default' for task execution.
[2022-06-17 07:40:57,054] {spark_submit.py:334} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077:7077 --conf spark.master=spark://spark:7077 --name Spark Hello World --verbose --queue root.default /usr/local/spark/app/hello-world.py /usr/local/spark/resources/data/***.cfg
[2022-06-17 07:40:57,163] {spark_submit.py:485} INFO - JAVA_HOME is not set
[2022-06-17 07:40:57,179] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 417, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark:7077:7077 --conf spark.master=spark://spark:7077 --name Spark Hello World --verbose --queue root.default /usr/local/spark/app/hello-world.py /usr/local/spark/resources/data/***.cfg. Error code is: 1.
[2022-06-17 07:40:57,184] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=spark-test, task_id=spark_job, execution_date=20220617T074054, start_date=20220617T074056, end_date=20220617T074057
[2022-06-17 07:40:57,202] {standard_task_runner.py:97} ERROR - Failed to execute job 3 for task spark_job (Cannot execute: spark-submit --master spark://spark:7077:7077 --conf spark.master=spark://spark:7077 --name Spark Hello World --verbose --queue root.default /usr/local/spark/app/hello-world.py /usr/local/spark/resources/data/***.cfg. Error code is: 1.; 550)
[2022-06-17 07:40:57,244] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-17 07:40:57,288] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
